lib/device_detector/device.rb

```diff
@@ -0,0 +1,54 @@
+class DeviceDetector
+  class Device < Parser
+
+    def known?
+      regex_meta.any?
+    end
+
+    def name
+      ModelExtractor.new(user_agent, regex_meta).call
+    end
+
+    def device_type
+      regex_meta['device']
+    end
+
+    private
+
+    def filenames
+      [
+        'devices/mobiles.yml'
+      ]
+    end
+
+    # Overwrite the default parser handling logic
+    def self.regexes_for(filepaths)
+      @regexes ||=
+        begin
+          regexes = YAML.load(filepaths.map { |filepath| File.read(filepath) }.join)
```

Comment:
Could you please make two lines out of this one? The `filepaths` joining would be more clear assigned to a separate variable.